doc/source/_templates/autosummary/no_methods_doc/base.rst.jinja2

```diff
@@ -0,0 +1,19 @@
+{{ objname | escape | underline}}
+
+.. currentmodule:: {{ module }}
+
+.. autoclass:: {{ objname }}
+
+   {% block attributes %}
+   {% if attributes %}
+   .. rubric:: {{ _('Attributes') }}
+
+   .. autosummary::
+      :toctree:
+   {% for item in attributes %}
+      {% if item not in inherited_members %}
```

Comment:
Did this if statement end up being required for the docs build to pass? The inherited members causing failure were methods, not attributes, right?

I'm mostly wondering whether we should merge this change to the `no_methods_doc` template in PyACP.

Reply:
This was needed because ints (and therefore IntEnums) have also a couple of properties (``denominator``, ``imag``, etc.), which have invalid docstrings. I guess it would make sense also change this also in pyACP. I opened a PR here: https://github.com/ansys/pyacp/pull/535